pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsData.scala

```diff
@@ -40,19 +43,22 @@ case class MetsData(
     reason = Some("METS work")
   )
 
-  private def digitalLocation(maybeLicense: Option[License]) = {
-    val url = s"https://wellcomelibrary.org/iiif/$recordIdentifier/manifest"
+  private def digitalLocation(license: Option[License],
```

Comment:
Why change the name? It's a convention used elsewhere?

Reply:
We use a mixture of both all over the place. Here the optional `license` field on `DigitalLocation` which we are assigning to isn't prefixed with a `maybe`, so naming it like this is consistent with that.

I'm not a big fan of using the `maybe` prefix in general, as that information is encoded within the type. Similarly, I wouldn't name all iterators or list types like `listWorks` / `iteratorWorks`, and just use `works`.

Reply:
> as that information is encoded within the type

I'm more than happy with that.